src/client/cli/cmd/start.cpp

```diff
@@ -59,8 +59,17 @@ mp::ReturnCode cmd::Start::run(mp::ArgParser* parser)
 
     auto on_success = [&spinner, this](mp::StartReply& reply) {
         spinner.stop();
+
         if (term->is_live() && update_available(reply.update_info()))
+        {
             cout << update_notice(reply.update_info());
+        }
+
+        if (reply.log_line() != "")
```

Comment:
maybe change `reply.log_line() != ""` to `!reply.log_line().empty()`.

Reply:
Right, changed.